src/icons/SealEye.tsx

```diff
@@ -1,9 +1,63 @@
-import { stroke } from 'classnames/tailwind'
+import classnames, { stroke, strokeWidth } from 'classnames/tailwind'
 
-const strokeSecondaryDimmed = stroke('stroke-secondary-dimmed')
-const strokeTertiaryLine = stroke('stroke-tertiary')
+enum SealEyeColor {
```

Comment:
Why do we use enum, when we can use `const SealEyeColor = 'stroke-accent' | 'stroke-secondary-dimmed' | 'stroke-tertiary'`?

Reply:
It's easier to make a mistake while typing `stroke-...`, so I set it to familiar names. I think it's more convenient